vms/platformvm/warp/README.md

```diff
@@ -0,0 +1,121 @@
+# Avalanche Warp Messaging
+
+Avalanche Warp Messaging (AWM) provides a primitive for Cross-Subnet Communication on the Avalanche Network.
```

Comment:
I think `Cross-Subnet Communication` should only be capitalized if it's a term that we'll be using a lot going forward and we want it to the name of a thing rather than an idea, if that makes sense.

Reply:
I think this should be `cross-subnet communication` since it's an idea rather than the thing that does it (Avalanche Warp Messaging.) Similarly I think `Warp Message` should be `Warp message` or `cross-subnet message`.

Reply:
Updating to `cross-subnet communication` and `warp message` except for where it's used as `Avalanche Warp Message`